pipeline.cpp

```diff
@@ -180,13 +223,20 @@ int main()
     while(running) 
     {           
         // Handle user inputs
-        processUserInputs(running);
+        //processUserInputs(running);
 
         // Refresh Screen
-        clearScreen(frame);
+        //clearScreen(frame);
 
+        /********** Your code goes here **********/
+        /* Uncomment to run the Game Of Life code *
+        GameOfLife(frame);
```

Comment:
Instead of keeping this here commented I would just remove these lines for elegance.